mobile/test/features/text_management/domain/use_cases/presentation/bloc/text_bloc_test.dart

```diff
@@ -121,20 +123,26 @@ void main() {
         title: tTitle,
         body: tBody,
         oldText: tText,
+        classroom: tClassroom,
```

Comment:
You should test with a new updatedClassroom, to check if the classroom was really updated